backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunServiceImpl.java

```diff
@@ -1362,6 +1362,7 @@ public void logArtifact(
       }
       Artifact artifact = artifacts.get(0);
 
+      artifactStoreDAO.initializeMultipart(artifact.getKey());
```

Comment:
Why are you calling this here? The client will explicitly initialize via the new API on getUrlForArtifact. I don't think we have to or should do it before.

Reply:
I think because the task says to add it to log artifact: https://vertaai.atlassian.net/browse/VR-4346 cc: @convoliution

Reply:
My proposal puts it in `logArtifact`, yes. 
If it's recommended to put it in `getUrlForArtifact` @conradoverta , then I can revise the document and tickets in the morning.

Reply:
Not every artifact will be uploaded. I'd recommend against creating a new flow if we don't have to. We can always revisit, but let's focus on delivering one thing at a time.

Reply:
moved initialization to getUrlForArtifact